src/one_dragon/base/screen/screen_loader.py

```
@@ -87,20 +91,23 @@ def reload(self, from_memory: bool = False, from_separated_files: bool = False)
-                file_path = os.path.join(self.yml_file_dir, file_name)
-                with open(file_path, 'r', encoding='utf-8') as file:
+                with file_path.open(encoding='utf-8') as file:
                     log.debug(f"加载yaml: {file_path}")
                     data = yaml_utils.safe_load(file)
                 if not isinstance(data, dict):
                     log.warning(f"画面配置格式错误，已跳过: {file_path}")
                     continue
 
                 screen_info = ScreenInfo(data)
+                if screen_info.screen_id in self._id_2_screen:
+                    log.warning(f"画面ID冲突，已跳过: {screen_info.screen_id}")
```

Ruff (0.15.13)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)